libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -27,6 +27,10 @@ set(compile_opts
   -sycl-std=2017
   )
 
+if (WIN32)
+	set(compile_opts "-D_ALLOW_RUNTIME_LIBRARY_MISMATCH ${compile_opts}")
```

Comment:
Looks like to many blanks.

Reply:
Also, I would use `list(APPEND)`, because `compile_opts` is a list.

Reply:
Done.
Thanks very much.